# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hyprland/sane-defaults.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Don't annoy new users about default config, or whine about devs firing it up in a VM as root.

--- Hyprland-0.51.0/src/main.cpp.vanilla	2025-09-10 16:11:10.358209092 +0200
+++ Hyprland-0.51.0/src/main.cpp	2025-09-10 16:11:16.188604199 +0200
@@ -55,7 +55,7 @@
     std::string configPath;
     std::string socketName;
     int         socketFd   = -1;
-    bool        ignoreSudo = false, verifyConfig = false;
+    bool        ignoreSudo = true, verifyConfig = false;
 
     if (argc > 1) {
         std::span<char*> args{argv + 1, sc<std::size_t>(argc - 1)};
--- hyprland-source/src/config/defaultConfig.hpp.vanilla	2024-11-23 14:30:19.433279957 +0100
+++ hyprland-source/src/config/defaultConfig.hpp	2024-11-23 14:31:57.139945039 +0100
@@ -9,7 +9,7 @@
 # OR EDIT THIS ONE ACCORDING TO THE WIKI INSTRUCTIONS.
 # #######################################################################################
 
-autogenerated = 1 # remove this line to remove the warning
+autogenerated = 0 # remove this line to remove the warning
 
 # This is an example Hyprland config file.
 # Refer to the wiki for more information.
